Auth_Handler.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Microsoft.Identity.Client;
+using Newtonsoft.Json.Linq;
+
+namespace Fabric_Metadata_Scanning
+{
+    class Auth_Handler
+    {
+        private static Auth_Handler instance = null;
+        public static Auth_Handler Instance
+        {
+            get
+            {
+                if (instance == null)
```

Comment:
This isn't a thread safe singleton pattern. If you need it to be thread safe please revise